modules.json

```diff
@@ -4,6 +4,9 @@
     "repos": {
         "https://github.com/nf-core/modules.git": {
             "modules": {
+                "nf-core": {}
+            },
+            "subworkflows": {
```

Comment:
These ones shouldn't exist, the diff shows that this _was_ on lines starting at 207. It should be moved back.